docs/nil/nil-101.mdx

```diff
@@ -17,7 +17,7 @@ The =nil; CLI is an easy-to-use tool for interacting with the cluster.
 To install the CLI:
 
 ```bash
-curl -fsSL https://github.com/NilFoundation/nil/raw/master/install.sh | bash
+curl -fsSL https://github.com/NilFoundation/nil/blob/main/scripts/install_cli.sh | bash
```

Comment:
thanks for noticing. old url seems to be revoked on the gh side. however this one doesn't work for all setups as well. I suggest to use `curl -fsSL https://github.com/NilFoundation/nil/raw/main/scripts/install_cli.sh`